include/SIMDswizzle.h

```diff
@@ -274,6 +274,12 @@ class IRR_FORCE_EBO SIMD_32bitSwizzleAble
 
 #define FAST_FLOAT_SHUFFLE(X,Y) _mm_castsi128_ps(_mm_shuffle_epi32(_mm_castps_si128(X),Y))
 
+#ifdef __GNUC__
+// warning: ignoring attributes on template argument �__m128i {aka __vector(2) long long int}� [-Wignored-attributes] (etc...)
```

Comment:
I thought we already had "Wnoerrr=ignored-attributes"

Reply:
`-Wno-error=ignored-attributes` doesn't take care of this?

Reply:
hm apparently not